app.js

```diff
@@ -41,63 +40,9 @@ const User = mongoose.model("User", userSchema);    //creating model according t
 // user.save();
 
 
-
-function findUserName(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.name);
-    }
-  })
-};
-
-
-
-
-function findUserEmail(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.email);
-    }
-  })
-};
-
-function findUserPassword(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.password);
-    }
-  })
-};
-
-
-function findUserId(name,email){
-  User.findOne({"Profile.email": email, "Profile.name": name}, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member._id);
-    }
-  })
-};
-
-function deleteUser(id_delete){
-  User.findByIdAndDelete(id,function(err){
-    if(err){
-      console.log(err);
-    }else{
-      console.log("The user was successfully deleted.");
-    }
-  })
-};
-id_d="";
-findUserName(id);
-findUserEmail(id);
-findUserPassword(id);
-findUserId(name,email);
-deleteUser(id_d);
+//UserScripts.createUser("Adrien", "myemail@gmail.com","safe_password" , User);
```

Comment:
Had to pass the User model to the functions as a result of placing them in the other file

Reply:
I actually thought it would be easier if we just left the models in the functions because I did not think we need those models in app.js but we definitely need them in UserScripts. Is there any reason we cannot leave them there?